skyportal/plot.py

```diff
@@ -1241,8 +1241,14 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
         else "box_zoom,wheel_zoom,pan,reset"
     )
     plot_width = None if device == "browser" else width
+    if device == "mobile_portrait":
+        aspect_ratio = 1
+    elif device == "tablet_portrait":
+        aspect_ratio = 1.5
+    else:
+        aspect_ratio = 1.8
     plot = figure(
-        aspect_ratio=2.0 if device == "mobile_landscape" else 1.5,
```

Comment:
It looks like `mobile_landscape` has become either `mobile_portrait` or `tablet_portrait`. Do we know if it's used anywhere else, and if not, maybe we should remove it?

Reply:
I believe it's still used in the photometry plot so I think we might as well keep it around here. I'd like to keep the option available in case future additions to these plots require more fine-tuned layout changes that require specifying something for `mobile_landscape` (same reason why I implemented both `tablet_portrait` and `tablet_landscape` originally even though we didn't actually have differences in the layouts for those two cases until this recent change in legend components)